sdks/python/apache_beam/coders/typecoders_test.py

```diff
@@ -140,6 +140,14 @@ def test_list_coder(self):
     self.assertIs(
         list, type(expected_coder.decode(expected_coder.encode(values))))
 
+  def test_nullable_coder(self):
+    expected_coder = coders.NullableCoder(coders.BytesCoder())
+    real_coder = typecoders.registry.get_coder(
+        typehints.UnionConstraint([type(None), type(bytes)]))
```

Comment:
nit: could you construct this with `Optional` instead of using `Union` directly?

Also I think `bytes` is already a type:
```py
>>> bytes
<class 'bytes'>
>>> type(bytes)
<class 'type'>
```

Shouldn't this be `Optional[bytes]`?

Reply:
Yep, I'll make the change in the test case